src/assets/json/communities.json

```diff
@@ -258,7 +258,42 @@
     "organizers": [
       {
         "name": "Michael Hladky",
-        "twitter": "https://twitter.com/michael_hladky"
+        "twitter": "https://twitter.com/michael_hladky",
+        "webs": ["https://www.linkedin.com/in/michael-hladky-519340148/"]
+      },
```

Comment:
1. Split the PR into two PRs due to 1PR add information about one community and the second PR is a new Community